packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -3,21 +3,25 @@
 :root {
   --terra-consumer-body-background: url('https://wallpapercave.com/wp/5ynt7Gb.jpg');
   --terra-consumer-nav-active-item-background: var(--terra-consumer-body-background);
+  --terra-consumer-mobile-panel-background: var(--terra-consumer-body-background);
+  --terra-consumer-layout-background: var(--terra-consumer-body-background);
   --terra-consumer-nav-item-color: #fff;
+  --terra-consumer-toggle-header-hover-color: var(--terra-consumer-nav-item-color);
 
-  --terra-consumer-nav-width: 320px;
+  --terra-consumer-mobile-panel-width: 320px;
   --terra-consumer-nav-text-color: var(--terra-consumer-nav-item-color);
 
-  --terra-consumer-profile-link-width: var(--terra-consumer-nav-width);
+  --terra-consumer-profile-link-width: var(--terra-consumer-mobile-panel-width);
   --terra-consumer-profile-link-background: var(--terra-consumer-body-background);
   --terra-consumer-profile-link-color: var(--terra-consumer-nav-item-color);
-  --terra-consumer-logo-text-color: var(--terra-consumer-nav-item-color);
+  --terra-consumer-nav-item-hover-color: var(--terra-consumer-nav-item-color);
+  --terra-consumer-profile-background: var(--terra-consumer-body-background);
 }
```

Comment:
do all these variables belong in layout? Alot of them are only used in nav specific component

Reply:
+1 We need to remove all the variables from layout and just have defaults set everywhere. If we want to sync their default value with sass variables, I'm OK with that.